modules/ROOT/pages/stats-monitoring.adoc

```diff
@@ -118,6 +118,90 @@ Global Sync Gateway stats
 
 Resource utilization stats
 
+===== admin_net_bytes_recv
+
+Number of bytes received on the admin interface.
+
+Throughput on admin interface = `admin_net_bytes_recv` / `admin_net_bytes_sent`
+
+===== admin_net_bytes_sent
+
+Number of bytes sent on the admin interface.
+
+Throughput on admin interface = `admin_net_bytes_recv` / `admin_net_bytes_sent`
+
+===== error_count
+
+Number of errors logged.
+
+===== go_memstats_heapalloc
+
+Go memstats.HeapAlloc
+
+===== go_memstats_heapidle
+
+Go memstats.HeapIdle
+
+===== go_memstats_heapinuse
+
+Go memstats.HeapInuse
+
+===== go_memstats_heapreleased
+
+Go memstats.HeapReleased
+
+===== go_memstats_pausetotalns
+
+Go memstats.PauseTotalNs
+
+===== go_memstats_stackinuse
+
+Go memstats.StackInuse
+
+===== go_memstats_stacksys
+
+Go memstats.StackSys
+
+===== go_memstats_sys
+
+Go memstats.Sys
+
+===== goroutines_high_watermark
+
+Peak number of go routines since process start.
+
+===== num_goroutines
+
+Number of goroutines.
+
+===== process_cpu_percent_utilization
+
+CPU utilization (%)
+
+===== process_memory_resident
+
+Memory utilization (Resident Set Size) for the process in bytes.
+
+===== pub_net_bytes_recv
+
+Number of bytes received on the public interface.
+
+Throughput on public interface = `pub_net_bytes_recv` / `pub_net_bytes_sent`
+
+===== pub_net_bytes_sent
+
+Number of bytes sent on the public interface.
+
+Throughput on public interface = `pub_net_bytes_recv` / `pub_net_bytes_sent`
+
+===== system_memory_total
+
+Total memory available on the system.
```

Comment:
@adamcfraser @bbrks is `system_memory_total` in bytes?

Reply:
yes

Reply:
https://github.com/couchbase/docs-sync-gateway/commit/052aca737a9fbc8087390c410065aea19b4e7299